### ISSUE TITLE
Enable tabular downloads of metadata through the search page

### ISSUE THREAD

**Issue by jagoldstein** [NCEAS/metacatui#534]
Prompted by [feedback](https://support.nceas.ucsb.edu/rt/Ticket/Display.html?id=16297) from Kate Stafford, we should develop a feature that allows users to download metadata in a tabular format. Similar to the Solr queries we perform on the backend, users ought to be able to search, select metadata fields of interest, and then download a file (CSV) listing the information of interest (identifiers, titles, etc.).

@csjx said that such a feature "certainly would be handy for PIs when they are doing their reports."

**Comment by mbjones** [comment]
Yeah, agreed.  The easiest thing may be to add a 'Download results as CSV'  option to the search results page, which just reformats the current search results and hands them off to the user.  No reason for a separate UI for this.

**Comment by mburrus** [comment]
ESS-DIVE is interested in a feature like this and is also interested in having this feature available in data portals. We're in agreement with @csjx 's earlier comment; this would have been very useful to have 2 weeks ago while some of our more active projects were compiling their annual reports.

Some additional fields not specifically mentioned above that we would like to see in such a table are:
- Project affiliation
- Publish date 
- Creation date
- Last Modified date
- Visibility (public vs private)
- The access level of the individual downloading the csv

Has there been discussion on this topic since the last comment was made in 2018?

**Comment by mbjones** [comment]
@mburrus No, we haven't further discussed or implemented this. Its still a good idea, but hasn't risen to high enough priority to make the cut compared to other features we are working on. Please chat with @csjx about where this fits in the ESS-DIVE priority list.
